src/components/Connection.tsx

```diff
@@ -148,6 +148,7 @@ const deleteindividualfiles=(index: number)=>{
     // Function to handle toggle for connect/disconnect button
     if (isConnected) {
       disconnectDevice();
+      deletedata();
```

Comment:
We should not delete the data when user disconnects, sometimes user can loose connection with port by mistake, in this case if there will be recordings they will get cleared. We can instead ask them if they want to store all data or delete it, and work accordingly.

Reply:
okay @akadeepesh  I will discuss it with sir, accordingly I'll make changes .

Reply:
we can show recorded files when we'll reconnect._